source/_top-nav.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav>
+  <ul class="top-nav">
+    <li class="top-nav__name"><span class="top-nav__name--impact">Jeff</span>Stolz</li>
+    <li class="top-nav__link"><%= link_to t("top-nav.work"), 'index.html' %></li>
+    <li class="top-nav__link"><%= link_to t("top-nav.about"), 'about.html' %></li>
+    <li class="top-nav__link"><%= link_to t("top-nav.contact"), 'contact.html' %></li>
```

Comment:
Similar note to the one above. Change those single quotes to double quotes:
`"index.html"`, etc